tests/prepare/feature/test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    for method in ${PROVISION_METHODS:-"virtual"}; do
+	# EPEL
+        rlPhaseStartTest "Enable EPEL"
+            rlRun -s "tmt run plan --name epel-enable provision --how $method prepare"
```

Comment:
You should use `tmt run --all` to actually run all steps, not just the `provision`. Have you run this?

Reply:
Sorry I didn't try before I was always thinking that the plugin is just related to `provision` and `prepare` phases. Today I gave it a try, the test results look like:

```bash
(tmt) huanli@KVM06:data$ cat plans.fmf
provision:
    how: container
    image: centos:stream9

execute:
    how: tmt
    script: bash -x ./check.sh

/epel-enabled:
    prepare:
        how: feature
        epel: enabled

# ...<snip>...

(tmt) huanli@KVM06:data$ cat check.sh
#!/bin/bash
exit 0

(tmt) huanli@KVM06:data$ tmt run --all \
                         plan --name epel-enabled \
                         provision --how container --image centos:stream8
/var/tmp/tmt/run-001

/plans/epel-enabled
    discover
        how: shell
        summary: 1 test selected
    provision
        how: container
        image: centos:stream8
        multihost name: default-0
        arch: x86_64
        distro: CentOS Stream 8
        summary: 1 guest provisioned
    prepare
        queued push task #1: push on default-0

        push task #1: push on default-0

        queued prepare task #1: default-0 on default-0

        prepare task #1: default-0 on default-0
        how: feature
        Enable EPEL on CentOS Stream 8
        Repo: epel is enabled.
        Repo: epel-debuginfo is enabled.
        Repo: epel-source is enabled.
        Repo: epel-next is enabled.
        Repo: epel-next-debuginfo is enabled.
        Repo: epel-next-source is enabled.

        queued pull task #1: pull on default-0

        pull task #1: pull on default-0

        summary: 1 preparation applied
    execute
        queued execute task #1: default-0 on default-0

        execute task #1: default-0 on default-0
        how: tmt
        progress:
        summary: 1 test executed
    report
        how: display
        summary: 1 test passed
    finish

        container: stopped
        container: removed
        summary: 0 tasks completed

total: 1 test passed
```